src/ansys/dpf/core/operators/utility/on_named_selection.py

```diff
@@ -35,7 +35,7 @@ class on_named_selection(Operator):
     >>> from ansys.dpf import core as dpf
 
     >>> # Instantiate operator
-    >>> op = dpf.operators.scoping.on_named_selection()
+    >>> op = dpf.operators.utility.on_named_selection()
```

Comment:
This change comes from an error server-side where this operator has been set as "hidden".